tests/params_test.py

```diff
@@ -1255,7 +1256,7 @@ def test_gy_uncertainty():
 
 mag_params = MagneticScalarParams(
     **{
-        "geoh5": "./FlinFlon.geoh5",
+        "geoh5": str(PROJECT),
```

Comment:
To be addressed on GEOPY-830, but Params should be able to take on `Path` type input for `geoh5` and `workspace_geoh5` values